### ISSUE TITLE
Use of expressions in custom model "multiply_by"

### ISSUE THREAD

**Issue by xallaume** [graphhopper/graphhopper#2743]
Hello,

The documentation at https://github.com/graphhopper/graphhopper/blob/master/docs/core/custom-models.md, in the chapter "The value expression", states that "The value of limit_to or multiply_by is usually only a number and can be a more complex expression like Math.sqrt(2)." and presents the following working example : "limit_to": "max_speed * 0.9".

But if we change the expression to be "limit_to": "max_speed / 0.9", the "/" operator isn't recognized. If we change it to "Math.sqrt(2)", it isn't recognized either.

**To Reproduce**

Steps to reproduce the behavior. For examples:

1. Go to https://graphhopper.com/maps/?point=42.31575%2C2.482717&point=42.322636%2C2.51102&profile=hike&layer=TF+Outdoors
2. Enter the following custom model in hike profile:
{
  "distance_influence": 15,
  "speed": [{
    "if": "true",
    "multiply_by": "max_speed / 0.9"  // or "Math.sqrt(2)"
  }],
  "priority": [],
  "areas": {}
}
3. You get a parsing error
4. The same happens in a local 6.2 deployment

**Expected behavior**
Use of expressions should be possible in "multiply_by" & "limit_to" statements as described in specs.

**System Information**

GraphHopper 6.2

Sincerely,

**Comment by easbar** [comment]
Yes, the maximum-speed estimation is used to improve A* performance (actually currently only for A*, not LM). For LM we have another limitation that is not needed for A*: We cannot increase speeds / reduce weights.
